src/deltachat.h

```diff
@@ -241,6 +241,8 @@ void            dc_openssl_init_not_required (void);
 // connect
 void            dc_configure                 (dc_context_t*);
 int             dc_is_configured             (const dc_context_t*);
+int             dc_is_imap_connected         (const dc_context_t*);
+int             dc_is_smtp_connected         (const dc_context_t*);
```

Comment:
is it a good idea to have the same name for a function that is defined differently in dc_configure.c and dc_smtp.c etc?

Reply:
I'm pretty happy with this naming btw, seems logical.  but let's see in the issue if this is PR is still desired I guess